tests/unit_tests/Commands/AliasesCommandTest.php

```diff
@@ -107,7 +106,7 @@ public function testAliases()
             ->with(
                 $this->equalTo('debug'),
                 $this->equalTo("Emitting aliases via {emitter}"),
-                $this->equalTo(['emitter' => 'Pantheon\Terminus\Helpers\AliasEmitters\AliasesDrushRcEmitter'])
+                $this->equalTo(['emitter' => 'Pantheon\Terminus\API\Helpers\AliasEmitters\AliasesDrushRcEmitter'])
```

Comment:
🤔 Perhaps these classes should be left in the CLI. The API itself doesn't consume them.

Reply:
I agree; the alias emitters are not API.